src/test/java/com/google/api/codegen/util/ScannerTest.java

```diff
@@ -70,6 +81,24 @@ public void testScannerDollar() {
     }
   }
 
+  @Test
+  public void testScannerInt() {
+    {
+      Scanner scanner = new Scanner("123 456");
```

Comment:
test also "0"

Reply:
and here or above, test also "57foo" (-> "57", "foo")

Reply:
Did you test something like "57foo"? We should capture the current behavior that parses it into two tokens.